spec/controllers/application_records_controller_spec.rb

```diff
@@ -75,6 +75,28 @@
     end
   end
 
+  describe 'GET #eeo_data' do
```

Comment:
I know you copied these test paradigms from your previous work, but still, nicely done